tests/utils/test_password_manager.py

```diff
@@ -3,6 +3,7 @@
 import os
 
 from typing import TYPE_CHECKING
+from unittest import mock
```

Comment:
We use `pytest-mock` package instead of `unittest`. It is a wrapper on unittest, but let's keep it consistent.

Reply:
Done @Secrus